src/Akka.Persistence.RavenDB/RavenDbConfiguration.cs

```diff
@@ -1,50 +1,79 @@
-﻿using Akka.Configuration;
+﻿using Akka.Actor;
+using Akka.Configuration;
 using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using System.Security.Cryptography.X509Certificates;
 
 namespace Akka.Persistence.RavenDb
 {
     public abstract class RavenDbConfiguration
     {
+        public const string CertificatePasswordVariable = "RAVEN_Security_Certificate_Password";
```

Comment:
let's revert this to `RAVEN_CERTIFICATE_PASSWORD`

Reply:
Done